src/Jackett.Common/Definitions/gay-torrentsorg.yml

```diff
@@ -24,6 +24,36 @@ settings:
     type: info
     label: "Search results"
     default: "You can increase the number of search results in your profile.<br />Default is 15."
+  - name: status
+    type: select
+    label: Status
+    default: 0
+    options:
+      0: All
+      1: Active Only
+      2: Dead Only
+      3: Free
+      4: 50%free
+      5: 25%free
+      6: packs
+  - name: sort
+    type: select
+    label: Sort by
+    default: seeds
+    options:
+      data: added
+      finished: downloads
+      seeds: seeders
+      leechers: leechers
+      size: size
+      filename: filename
```

Comment:
limit to added, filename, seeders, and size (and rename added to created, and filename to title)